### DISCUSSION TITLE
Is rebar3 dependency "git_subdir" tag supported ?

### DISCUSSION THREAD

**Discussion by netboz** [inaka/elvis#581] [Q&A] [answered]
Hi,

In my rebar.config, I have something like this :

`{deps,
 [....
  {opentelemetry_api,
   {git_subdir,
    "http://github.com/open-telemetry/opentelemetry-erlang",
    {branch, "main"},
    "apps/opentelemetry_api"}},
....]).`

Elvis is reporting an error at the head f my source file : 

`'function_clause' while applying rule 'protocol_for_deps_rebar'.Elvis(Config Error)`

I was so wondering if `git_subdir` was supported ( I have found nothing regarding this into the code and googling ).

Thanks for any answer

**Comment by elbrujohalcon** [comment, marked as answer]
It is not… and that's a bug. I'll label this ticket as such. Thanks for the report.

**Comment by elbrujohalcon** [comment]
Oh, this is a convo… well… I'll turn it into a ticket :)

**Comment by elbrujohalcon** [comment]
This is now #582 :)

**Comment by netboz** [comment]
Thanks a lot for keeping Elvis rocking :)
